shell/edit/fleet.cattle.io.gitrepo.vue

```diff
@@ -601,7 +603,21 @@ export default {
         </div>
       </template>
       <div class="spacer" />
+      <h2 v-t="'fleet.gitRepo.resources.label'" />
 
+      <Banner
```

Comment:
I think we should modify the Banner text - I think the text should be clearer that the resource handling only happens when the checkbox is checked- suggest:

`When enabled below, resources will be kept when deleting a GitRepo or Bundle - only Helm release secrets will be deleted.`

Reply:
@kwwii see comment above - at present it looks like this:

![image](https://user-images.githubusercontent.com/1955897/232842989-786c0579-b37a-4bfb-9c61-78815ac1f271.png)

The usual pattern is to have the content as part of the checkbox, under the checkbox label - but at least being clearer that this is the behaviour when checked seems important.

Reply:
Agreed, the text should read:
"When enabled, resources will be kept when deleting a GitRepo or Bundle - only Helm release secrets will be deleted"
and the info banner should be placed below the checkbox as it is elsewhere in the UI